apps/judicial-system/backend/src/app/modules/notification/test/notificationController/getAllCaseNotificationsRolesRules.spec.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  judgeRule,
+  prosecutorRule,
+  registrarRule,
+  representativeRule,
+  assistantRule,
+} from '../../../../guards'
+import { NotificationController } from '../../notification.controller'
+
+describe('NotificationController - Get all case notifications rules', () => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let rules: any[]
+
+  beforeEach(() => {
+    rules = Reflect.getMetadata(
+      'roles-rules',
+      NotificationController.prototype.getAllCaseNotifications,
+    )
+  })
+
+  it('should give permission to five roles', () => {
```

Comment:
Consider changing the description of this test to be a little less specific, something like "should give permission to appropriate roles". That way we don't need to update the title if roles are added or removed.

Reply:
Good point. Will keep in mind for the future, but there is a huge number of test with this kind of descriptions 😅